composite.c

```diff
@@ -314,6 +331,20 @@ usb_device_composite_struct_t* composite_init(userCbFunc callback, void* userArg
             LOG_ERROR("[Composite] VirtualCom initialization failed");
     }
 
+    #if defined (USB_DEVICE_CONFIG_USE_TASK) && (USB_DEVICE_CONFIG_USE_TASK > 0U)
+    if (xTaskCreate(USB_DeviceTask,                  /* pointer to the task */
+                    (char const *)"usb device task", /* task name for kernel awareness debugging */
+                    3072L / sizeof(portSTACK_TYPE),  /* task stack size */
+                    composite.deviceHandle,          /* optional task startup argument */
+                    tskIDLE_PRIORITY,                /* initial priority */
+                    &composite.device_task_handle    /* optional task handle to create */
+                    ) != pdPASS)
+    {
+        LOG_ERROR("[Composite] Failed to create usb device task!");
+        return NULL;
+    }
+    #endif
```

Comment:
Please do not run threads which are doing nothing while USB not connected. It add substantial CPU load which is especially important for Bell.

Reply:
You're right, this has to be refactored further. This is a kind of first quick fix to the issue.

Reply:
Actually, this task is needed as is. In principle, this task sleeps most of the time on OSA_MsgQGet() to get a message put by USB_OTG1 IRQ handler.

Reply:
It will be great if you could measure if any cpu overhead occurs.